### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by asiajanicka**

### CODE REVIEW

File: src/main/java/org/example/Utils.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        JavascriptExecutor je = (JavascriptExecutor) driver;
+        long windowWidth = (long) je.executeScript("return window.innerWidth");
+
+        int elX = el.getLocation().getX();
+        int elWidth = el.getSize().getWidth();
+
+        return ((elX + elWidth) <= (windowWidth / 2));
+    }
+
+    public static boolean isCurrentPageLoaded(WebDriver driver) {
```
Comment (vboltrukanis) [inline src/main/java/org/example/Utils.java]: default pageLoading methods didn't work?

File: src/test/java/org/example/CarinaWebTests.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        String expectedHeadingText = "Overview";
+
+        CucumberPage cucumberPage = new CucumberPage(getDriver());
+        homePage = cucumberPage.goToPage().getHeader().clickLogo();
+
+        assertThat(homePage.getContentBlock().getHeadingText()).isEqualTo(expectedHeadingText);
+    }
+
+    @Test
+    public void carinaLabelIsOnHeaderTest() {
```
Comment (vboltrukanis) [inline src/test/java/org/example/CarinaWebTests.java:42]: you should combine this simple tests if possible. Bcs you will significantly increase execution time of your suite with test starting drover session and opening page. There should be some balance, and we can validate ui of several components in one test.

**Review by vboltrukanis — APPROVED** [review]

File: src/main/java/org/example/Utils.java (hunk 1 above)
Comment (asiajanicka) [reply src/main/java/org/example/Utils.java]: changed. now i used wait.until.

File: src/test/java/org/example/CarinaWebTests.java (hunk 2 above)
Comment (asiajanicka) [reply src/test/java/org/example/CarinaWebTests.java:42]: done

**Pull request merged**